migration.md

```diff
@@ -0,0 +1,63 @@
+# Questions
+
+This document needs to be cleaned but I thought this might be better than an infinate issue thread 😉
+
+## General
+- I see a lot of `fs` usage in the [with-mdx-remote example](https://github.com/vercel/next.js/tree/canary/examples/with-mdx-remote)
+  should this worry me? (as far as I know this is not a browser-supported package and only works locally)
```

Comment:
No, any imports used in `getStaticProps` are executed in a node environment at build time and removed from client side output. The [nextjs docs](https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation) can offer more detail on this.